src/pages/Delivery/Campus/index.tsx

```diff
@@ -0,0 +1,280 @@
+                    onClick={() => {
+                      if (isSelected) {
+                        setSelectedCampusCategory(null);
+                        return;
+                      }
+                      setSelectedCampusCategory(campus.name);
+                    }}
+                  >
+                    <div className="flex items-center gap-2">
+                      <span className="text-primary-500 text-sm">{campus.name}</span>
+                      {campus.icon && <campus.icon />}
+                    </div>
+                    <span className={clsx('transition-transform', isSelected && 'rotate-180')}>
+                      <ChevronDown />
+                    </span>
+                  </button>
+                </div>
+                {isSelected && (
+                  <>
+                    <div className="h-[1px] w-[310px] bg-[#eee]" />
+                    <div className="mx-auto flex w-full flex-wrap justify-center gap-x-3 gap-y-2 bg-white px-4 py-4 break-all">
+                      {campus.building.map((building) => (
+                        <Badge
+                          key={building}
+                          label={building}
+                          color={selectedCampusBuilding === building ? 'primary' : 'neutral'}
+                          size="lg"
+                          onClick={() => setSelectedCampusBuilding(building)}
+                        />
+                      ))}
+                    </div>
+                  </>
+                )}
+              </div>
+            );
+          })}
+        </div>
+      </div>
+
+      <div className="mt-[1.813rem]">
+        <div className="text-primary-500 pb-2 font-semibold">배달기사님에게</div>
+        <button
+          type="button"
+          onClick={handleOpenBottomModal}
+          className="flex w-[21.375rem] items-center justify-between rounded-sm border border-neutral-300 bg-white px-4 py-3 text-sm text-neutral-400"
+        >
+          {requestLabel()}
+          <div className="pointer-events-none">
+            <ArrowDown />
+          </div>
+        </button>
+      </div>
+      <Button className="mt-[18.188rem] h-[2.875rem] w-[21.375rem]" onClick={handleOpenModal}>
+        주소 선택
+      </Button>
+      <Modal isOpen={modalIsOpen} onClose={handleCloseModal}>
+        <ModalContent>정확한 상세 주소를 입력해주세요.</ModalContent>
+        <ModalFooter>
+          <Button onClick={handleCloseModal} className="h-12 w-[16rem]">
+            확인
+          </Button>
+        </ModalFooter>
+      </Modal>
+      <BottomModal isOpen={bottomModalIsOpen} onClose={handleCloseBottomModal}>
+        <BottomModalHeader>
+          <div className="text-primary-500 font-semibold">배달기사님에게</div>
+          <button onClick={handleCloseBottomModal}>
+            <CloseIcon />
+          </button>
+        </BottomModalHeader>
+        <BottomModalContent>
```

Comment:
```suggestion
        <BottomModalContent className="px-6">
```
BottomModal이 사용되는 곳에 따라 내부 패딩 값이 조금 차이나는 부분이 존재해서 그대로 사용하면 UI가 어긋나는 부분이 존재해요 여기서는 px-6을 주면 디자인과 동일하게 될 것 같아요!